### PR TITLE
New york Times project for microverse

### PR DESCRIPTION
**Pull request by dasileker**
### Project floating Elements  For microverse 

@TSHEPO-CLOUD  &  @dasileker 

> in this project we worked with :

- HTML & CSS 

- images as svgs

- Iframe For video

- Aside tag &  grid tag 

-  floating elements

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1091 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>New York Times</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <nav class="first-nav">
+    <div class="menu-icon">
+      <img src="images/menu.png" alt="Menu">
+    </div>
```
Comment (peterrobert) [inline index.html]: This Hamburger icon should be a button. You can do this by using the button tag

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle:  :stop_sign: 
### HELLO @dasileker  AND @TSHEPO-CLOUD  VERY NICE WORK ON THIS PROJECT. YOU HAVE DONE A GREAT JOB. THERE ARE JUST SOME FEW CHANGES TO BE MADE KINDLY WORK ON THE FOLLOWING:

### [OPTIONAL]
There are so many warnings in your HTML. Try and fix them. This is however not a requirement since they are not errors.

Kindly fix that one required change below :arrow_down: 

### if you have any question kindly reach me on slack @peterrobert

### HAPPY CODING :computer:

File: index.html (hunk 1 above)
Comment (dasileker) [reply index.html]: okay thanks for the review I did change the IMG with a button I hope it's good

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,1055 @@
...
+  z-index: 10;
+  box-sizing: border-box;
+}
+
+.menu-icon button {
+  width: 50px;
+  height: 30px;
+  margin-top: 20px;
+  background-color: white;
+  border: 1px solid grey;
```
Comment (kitpao) [inline css/style.css:164]: - [ ] **OPTIONAL** Your border seems a bit off, you could take it out with `border: none;`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1091 @@
...
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>New York Times</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <nav class="first-nav">
+    <div class="menu-icon">
+      <button type="submit">&#9776;</button>
```
Comment (kitpao) [inline index.html]: - [x] **MANDATORY** You are failing your linter errors, this special characters often trigger them. As a suggestion, I would recommend you to use [Font awesome](https://fontawesome.com/v4.7.0/icon/bars) for your icons, or use a svg/image :)

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello @dasileker and @TSHEPO-CLOUD 

You did a pretty good job! 🥇 Your project is well detailed and the positioning for your elements is excellent! 💯

~~You are only missing one mandatory change, please check below~~

### ~~Status: Your project requires some additional changes~~

**Note:** Please submit for review only after you are sure you already finished 😮 I almost submitted it to the system
You only have one **optional** suggestion below ⬇️ 

### 👏 Your project has been approved 👏
### 🚀 You may merge to master and submit the project completition form 🚀

___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

File: index.html (hunk 3 above)
Comment (dasileker) [reply index.html]: sorry I just pushed the file again I will update the demo link

**Pull request merged**